src/repository/screens/repository.screen.js

```diff
@@ -118,6 +118,14 @@ class Repository extends Component {
     }
   };
 
+  refreshRepo = () => {
+    const { navigation } = this.props;
```

Comment:
Same thing here. Maybe rename this to something like `getRepoInfo` and call the method within `componentDidMount` and `refreshControl` prop so we can avoid duplication